src/geometries/ParametricGeometry.js

```diff
@@ -72,6 +77,16 @@ function ParametricBufferGeometry( func, slices, stacks ) {
 			var p = func( u, v );
 			vertices.push( p.x, p.y, p.z );
 
+			// approximate tangent plane vectors via central difference
+
+			pu.subVectors( func( u + EPS, v ), func( u - EPS, v ) );
+			pv.subVectors( func( u, v + EPS ), func( u, v - EPS ) );
```

Comment:
Sounds reasonable. We can also adjust some entities in `THREE.ParametricGeometries` so they use the `optionalTarget`. But let's do this in a separate PR, okay?

@jaxry Are you interested in doing this 😉 ?